string-manipulation-2/src/num-vowels.js

```diff
@@ -0,0 +1,12 @@
+/* exported numVowels */
+function numVowels(string) {
+  string = string.toLowerCase();
+  string = string.split('');
```

Comment:
You can loop through strings without splitting them, since they are indexed and have a `length` property.